rosdep/base.yaml

```diff
@@ -1317,20 +1321,20 @@ libcoin60-dev:
   arch: [coin]
   debian: [libcoin60-dev]
   fedora: [Coin2-devel]
-  gentoo: [media-libs/coin]
+  gentoo: ['=media-libs/coin-3.1.3*']
```

Comment:
Also not sure about this version number -- how did you get from coin60 to coin-3.1.3?